robottelo/ssh.py

```diff
@@ -117,6 +117,7 @@ def get_client(
         password=password,
         timeout=timeout,
         port=port,
+        allow_agent=False if password else True,
```

Comment:
One day this module might be gone too!

Reply:
oh that is going to be another juicy change!